src/test/clientApp/src/main/java/org/lfenergy/operatorfabric/client/cards/CardData.java

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+package org.lfenergy.operatorfabric.client.cards;
+
+import org.lfenergy.operatorfabric.cards.model.Card;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CardData {
+    Card card;
+    Card[] childCards;
+}
```

Comment:
I'm not sure you should create a CardData class like that as there would be nothing enforcing the fact that it should stay aligned with the one under [cards-publication](https://github.com/opfab/operatorfabric-core/tree/develop/services/core/cards-publication/src/main/java/org/lfenergy/operatorfabric/cards/publication/model), which is precisely what the whole swagger-generated interfaces (for all the pains that it causes...) was meant to do.
I would have expected it to follow the usual pattern of swagger def -> generates concrete class under client and matching interface under services... It's my bad because I merged Samy's PR that introduced it... @freddidierRTE what do you think?

Reply:
Created https://opfab.atlassian.net/browse/OC-1516 to address this.